solver/src/interactions.rs

```diff
@@ -2,4 +2,23 @@
 pub mod dummy_web3;
 mod uniswap;
 
+use crate::encoding;
+use anyhow::anyhow;
+use anyhow::Result;
+use primitive_types::H160;
 pub use uniswap::UniswapInteraction;
+use web3::types::Bytes;
+
+fn encode_interaction(
+    target: H160,
+    calldata: Bytes,
```

Comment:
I prefer Vec<u8> directly instead of web3::Bytes because I don't feel the wrapper adds anything.